src/main/java/uk/gov/hmcts/darts/usermanagement/service/validation/UserTypeValidator.java

```diff
@@ -22,8 +22,7 @@ public void validate(Integer userId) {
 
         if (accountEntity.isPresent() && accountEntity.get().getIsSystemUser()) {
             throw new DartsApiException(
-                UserManagementError.USER_NOT_FOUND,
-                String.format("User id %d is a system user", userId));
+                UserManagementError.USER_NOT_FOUND);
```

Comment:
I'd suggest we return the same error message that the existing USER_NOT_FOUND error uses, so the two are indistinguishable, i.e. [this](https://github.com/hmcts/darts-api/blob/8cd57fb5fe5d04bbcd6e06edfb97a6436abab953/src/main/java/uk/gov/hmcts/darts/usermanagement/service/validation/UserAccountExistsValidator.java#L21) message

Reply:
@chrisbellingham-hmcts ?

Reply:
Agree, motivation for this change to keep the behaviour consistent so let's include the `String.format("User id %d not found", userId));` element.